invite/app.ts

```diff
@@ -15,28 +15,109 @@
  */
 
 import {Application, Context} from 'probot';
+import {createTokenAuth} from '@octokit/auth';
+import Webhooks from '@octokit/webhooks';
+import {Octokit} from '@octokit/rest';
 
-module.exports = (app: Application) => {
+import {InviteBot} from './src/invite_bot';
+
+type WebhookHandler<T> = (inviteBot: InviteBot, payload: T ) => Promise<void>;
+
+type CommentWebhookPayload =
+  | Webhooks.WebhookPayloadIssues
+  | Webhooks.WebhookPayloadIssueComment
+  | Webhooks.WebhookPayloadPullRequest
+  | Webhooks.WebhookPayloadPullRequestReview
+  | Webhooks.WebhookPayloadPullRequestReviewComment;
+
+export default (app: Application) => {
```

Comment:
One day I'll learn the difference between `module.exports` and `export` and `export default`

Reply:
`module.exports` is the ES6/node standard way
`export foo` --> `const {foo} = require('...')`
`export default foo` --> `const foo = require('...')`